test/test-projects/junit/src/test/java/junit4/ExampleTest.java

```diff
@@ -0,0 +1,35 @@
+package junit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class ExampleTest {
```

Comment:
I think we can keep one of `junit4/ExampleTest` and `junit4/ParameterizedTest` since their content looks very similiar.

Maybe use `junit4/ExampleTest` to replace ``junit4/ParameterizedTest``?